### DISCUSSION TITLE
Is there a way to report more properties to MQTT, like charge_port_latch

### DISCUSSION THREAD

**Discussion by ybizeul** [teslamate-org/teslamate#1266] [Q&A]
I'm interested in reporting `charge_port_latch` (is the charger port locked) or HVAC state but it doesn't seem teslamate is sending this to mqtt. Am I missing some way to customize what is notified ?

**Comment by LelandSindt** [comment]
Along these lines I have some automation that I would like to trigger depending on the state of `charge_port_door_open`

I poured over the code base but I just couldn't grok where I could add additional parameters. @adriankumpf If you could point me in the right direction I would be glad to make an attempt at adding these parameters.

**Reply by LelandSindt** [reply]
It occurred to me this morning that a well formed git commit would point me in the right direction..

https://github.com/adriankumpf/teslamate/commit/3d13056bdf60b97dac4b4eb1b538543ed305b53b https://github.com/adriankumpf/teslamate/commit/85eb9307e2bfd8db9f87dada45b6c06bd6e95689

--- thank you for keeping your commits concise. :smile:

**Reply by LelandSindt** [reply]
well, I feel quite the fool... `charge_port_door_open` _is already published_ to mqtt...
